sharedbuffers/mapped_struct.py

```diff
@@ -1969,25 +1989,28 @@ def _subset(self, seq, strict_subset):
                         not strict_subset or self.bitlen < pfset.bitlen)
                 else:
                     dcode, objlen, itemsize, offset, _struct = self.objlist._metadata()
+                    if dcode not in ('q', 'I', 'i', 'd'):
```

Comment:
`I` and `i` are numeric... or did you mean to say `t` and `T`?

Reply:
```dcode in ('t', 'T')``` is semantically equivalent to ```dcode not in ('q', 'I', 'i', 'd')```
Since I used the latter tuple below, I figured it would be more clear this way.